styles/alert.css

```diff
@@ -22,3 +22,102 @@
   --bs-alert-color: rgb(var(--stanford-illuminating-dark-rgb));
   --bs-alert-border-color: rgb(var(--stanford-illuminating-dark-rgb));
 }
+
+/* Banner styles */
+.banner {
+  --banner-bg: none;
+  --banner-padding-x: 16px;
+  --banner-padding-y: 16px;
+  --banner-margin-bottom: 1rem;
+  --banner-color: inherit;
+  --banner-border-color: transparent;
+  --banner-border: var(--bs-border-width) solid var(--banner-border-color);
+  --banner-border-radius: none;
+  --banner-link-color: inherit;
+
+  position: relative;
+  padding: var(--banner-padding-y) var(--banner-padding-x);
+  margin-bottom: var(--banner-margin-bottom);
+  color: var(--banner-color);
+  background-color: var(--banner-bg);
+  border: var(--banner-border);
+  border-radius: var(--banner-border-radius);
+  border-left: 4px solid var(--banner-border-color);
+  border-top: none;
+  border-right: none;
+  border-bottom: none;
+}
+
+.banner-primary {
+  --banner-color: var(--bs-primary-text-emphasis);
+  --banner-border-color: var(--bs-primary-border-subtle);
+  --banner-link-color: var(--bs-primary-text-emphasis);
+}
+
+.banner-secondary {
+  --banner-color: var(--bs-secondary-text-emphasis);
+  --banner-border-color: var(--bs-secondary-border-subtle);
+  --banner-link-color: var(--bs-secondary-text-emphasis);
+}
+
+.banner-success {
+  --banner-color: var(--bs-success-text-emphasis);
+  --banner-border-color: var(--bs-success-border-subtle);
+  --banner-link-color: var(--bs-success-text-emphasis);
+}
+
+.banner-info {
+  --banner-color: var(--bs-info-text-emphasis);
+  --banner-border-color: var(--bs-info-border-subtle);
+  --banner-link-color: var(--bs-info-text-emphasis);
+}
+
+.banner-note {
+  --banner-color: rgb(var(--stanford-illuminating-dark-rgb));
+  --banner-border-color: rgb(var(--stanford-illuminating-dark-rgb));
+  --banner-link-color: var(--stanford-illuminating-dark-rgb);
+}
+
+.banner-warning {
+  --banner-color: var(--bs-warning-text-emphasis);
+  --banner-border-color: var(--bs-warning-border-subtle);
+  --banner-link-color: var(--bs-warning-text-emphasis);
+}
+
+.banner-danger {
+  --banner-color: var(--bs-danger-text-emphasis);
+  --banner-border-color: var(--bs-danger-border-subtle);
+  --banner-link-color: var(--bs-danger-text-emphasis);
+}
+
+.banner-light {
+  --banner-color: var(--bs-light-text-emphasis);
+  --banner-border-color: var(--bs-light-border-subtle);
+  --banner-link-color: var(--bs-light-text-emphasis);
+}
+
+.banner-dark {
+  --banner-color: var(--bs-dark-text-emphasis);
+  --banner-border-color: var(--bs-dark-border-subtle);
+  --banner-link-color: var(--bs-dark-text-emphasis);
+}
+
+.banner h3 {
+  padding-right: var(--banner-padding-x);
+
+  .heading-tag {
```

Comment:
I also don't see this anywhere which means --banner-link-color doesn't need to be set.